tests/phpunit/unit/Controller/FrontendTest.php

```diff
@@ -405,6 +480,20 @@ public function testSearchListing()
 
         $response = $this->controller()->search($this->getRequest());
 
+        $this->assertTrue($response instanceof TemplateView);
+        $this->assertSame('search.twig', $response->getTemplate());
+    }
+
+    /**
+     * @group legacy
+     */
+    public function testSearchLegacy()
```

Comment:
I'm voting for the annotation in the end, as:
  * Consistency
  * `phpunit --filter testSearch` will run both, so if I am debugging a change I get the legacy test running without thinking about it

But "tested" the name change and it does as you say, and I think that's freaking cool nonetheless!